flterm.c

```diff
@@ -1110,7 +1115,7 @@ int main(int argc, char *argv[])
 	}
 
 	/* Banner */
-	printf("[FLTERM] Starting...\n");
+	printf("[FLTERM] v2.8.2 Starting...\n");
```

Comment:
I'm not fond of this hard coded version number here. It is quickly going to be lieing to people.

If you remove this bit, I will merge the ctrl-d and add an auto-generated version info.

Reply:
Turns out there is already a GIT_VERSION macro;
```suggestion
	printf("[FLTERM] " GIT_VERSION " Starting...\n");
```

Reply:
Turns out I can just do this to your branch....